labelbox/schema/project.py

```diff
@@ -352,6 +352,47 @@ def create_prediction(self, label, data_row, prediction_model=None):
         res = self.client.execute(query_str, params)
         return Prediction(self.client, res["createPrediction"])
 
+    def upload_annotations(
+            self,
+            name: str,
+            url: str = None,
+            path: str = None,
+            annotations: Iterable[dict] = None
```

Comment:
In general, imho  separate methods are much easier to understand than exclusive args. 
<Java mode on>
How about having an internal AnnotationUploader class with methods `from_url`, `from_path`, `from_object`? Then you'd do `self.annotation_uploader(name).from_url(url)`
<Java mode off>

Reply:
updated